build.gradle

```diff
@@ -0,0 +1,112 @@
+
+// This gradle build depends on Mendix internal resources, the lastest commit
+// which did not is 6e1527a5e31347c6914bc34deab31ef4ab46953b. You may restore
+// the gradle scripts to that commit if you which to use them at your own risk.
+// We don't guarantee that those will keep working in the future.
+
+import com.github.jk1.license.render.*
+
+plugins {
+    id 'java'
+    id 'com.mendix.gradle.publish-module-plugin' version '1.6'
+    id 'net.researchgate.release' version '2.8.1'
+    id 'com.github.jk1.dependency-license-report' version '2.0'
+}
+
+sourceCompatibility = '11'
+[compileJava, compileTestJava]*.options*.encoding = 'UTF-8'
+
+project.ext {
+    mendixPublicApiVersion = '8.12.5'
+}
+
+mxMarketplace {
+    appNumber = 37114
+    appName = 'ObjectHandling'
+    moduleName = 'CommunityCommons_ObjectHandling'
+    moduleLicense = 'Apache V2'
+    appDirectory = "src/ObjectHandling"
+    versionPathPrefix = "_Version " // the path prefix within the module to the version folder
```

Comment:
I think this should be `"_Docs/version "`

Reply:
You're right, it is stored in a slightly different location than CommunityCommons.

Reply:
The space is significant (after the `version`). You can test it with `gradle moduleVersion`. Notice that with your current change (without the space) it will print the version number with a space before it, I'm afraid that may cause further issues.

Reply:
Fixed.